### PR TITLE
3주차

### PR DESCRIPTION
**Pull request by pumpkiinbell**

### CODE REVIEW

File: pumpkiinbell/chapter4.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+### 제어 장치 (Control Unit)
+
+![control-unit](https://www.elprocus.com/wp-content/uploads/control-unit-block-diagram.jpg)
+
+**제어 장치가 받아들이는 정보**
+
+- 클럭(Clock) 신호
+  - Clock 은 컴퓨터의 모든 부품을 일괄적으로 움직일 수 있게 하는 시간 단위이다.
+    - 가령, 클락의 주기에 맞춰 레지스터 간의 데이터 이동이나 ALU 연산 수행 등으 ㄹ하게 된다.
```
Comment (yoreleihee) [inline pumpkiinbell/chapter4.md:30]:
```suggestion
    - 가령, 클락의 주기에 맞춰 레지스터 간의 데이터 이동이나 ALU 연산 수행 등을 하게 된다.
```

File: pumpkiinbell/chapter4.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  - 플래그 레지스터 속 플래그 값을 받는다.
+- 시스템 버스, 그 중 제어 버스로 전달된 제어 신호를 받는다.
+  - 제어 신호는 CPU 외부 장치도 발생시킬 수 있다.
+
+**제어 장치가 내보내는 정보**
+
+- CPU 외부에 전달하는 제어 신호
+  - 즉, 제어 버스로 제어 신호를 내보낸다는 말과 같다.
+  - 메모리, 입출력 장치에 전달하는 제어 신호가 있다.
+    - 메모리에 저장된 값을 일걱나 새로운 값을 쓰고 싶다면 메모리로 제어 신호를 내보낸다.
```
Comment (yoreleihee) [inline pumpkiinbell/chapter4.md:44]:
```suggestion
    - 메모리에 저장된 값을 읽거나 새로운 값을 쓰고 싶다면 메모리로 제어 신호를 내보낸다.
```

File: pumpkiinbell/chapter4.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+### 특정 레지스터를 이용한 주소 지정 방식(1): 스택 주소 지정 방식
+
+**스택 주소 지정 방식**
+
+- 스택과 스택 포인터를 이용한 주소 지정 방식이다.
+
+**스택 포인터**
+
+- 스택 주소 지정 방식으로 으로 주소를 지정할 때 사용한다.
```
Comment (yoreleihee) [inline pumpkiinbell/chapter4.md:93]:
```suggestion
- 스택 주소 지정 방식으로 주소를 지정할 때 사용한다.
```

**Review by yoreleihee — APPROVED** [review]

**Comment by pumpkiinbell** [comment]
@yoreleihee 오탈자 수정했습니다~ https://github.com/haedal-study/self-learning-comp-archi/pull/23/commits/0944c385b39ca24ff14e781a1b9c16c35a6c1d43

**Pull request merged**